tests/models_tests/test_mlp.py

```diff
@@ -0,0 +1,62 @@
+from chainer import cuda
+from chainer import gradient_check
+import numpy
+import pytest
+
+from chainer_chemistry.config import MAX_ATOMIC_NUM
+from chainer_chemistry.models.mlp import MLP
+
+atom_size = 5
+out_dim = 4
+batch_size = 2
+
+
+@pytest.fixture
+def model():
+    return MLP(out_dim=out_dim)
+
+
+@pytest.fixture
+def data():
+    atom_data = numpy.random.randint(
+        0, high=MAX_ATOMIC_NUM, size=(batch_size, atom_size)
+    ).astype(numpy.float32)
```

Comment:
input of MLP is expected to be a hidden feature vector (=random float value), instead of embed vector (int value of atomic number).